src/services/subject.js

```
@@ -43,6 +44,28 @@ const subjectService = {
     }
 
     return subject;
+  },
+
+  updateSubject: async (subjectId, updateData) => {
+    const subject = await Subject.findById(subjectId);
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)